pom.xml

```diff
@@ -1,133 +1,213 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
-    <parent>
-        <groupId>io.openshift.booster</groupId>
-        <artifactId>spring-boot-booster-parent</artifactId>
-        <version>1.5.13-2</version>
-    </parent>
-
-    <artifactId>spring-boot-istio-distributed-tracing-parent</artifactId>
-    <version>1.5.13-3-SNAPSHOT</version>
-    <packaging>pom</packaging>
-
-    <name>Spring Boot - Istio Distributed Tracing Booster</name>
-    <description>Spring Boot - Istio Distributed Tracing Booster</description>
-
-
-    <properties>
-        <java.version>1.8</java.version>
-
-        <skip.fmp>true</skip.fmp>
-
-        <!-- Default values  / used when no profile is activated-->
-
-        <greeting-service.port>8090</greeting-service.port>
-        <cute-name-service.port>8091</cute-name-service.port>
-
-        <cute-name-service.endpoint>localhost:8091</cute-name-service.endpoint>
-
-        <jaeger.endpoint>http://localhost:14268/api/traces</jaeger.endpoint>
-    </properties>
-
-    <modules>
-        <module>spring-boot-istio-distributed-tracing-greeting-service</module>
-        <module>spring-boot-istio-distributed-tracing-cute-name-service</module>
-    </modules>
-
-    <build>
-        <resources>
-            <resource>
-                <directory>src/main/resources</directory>
-                <filtering>true</filtering>
-            </resource>
-        </resources>
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>io.openshift</groupId>
+    <artifactId>booster-parent</artifactId>
+    <version>22</version>
+  </parent>
+
+  <artifactId>spring-boot-istio-distributed-tracing-parent</artifactId>
+  <version>1.5.13-3-SNAPSHOT</version>
+  <packaging>pom</packaging>
+
+  <groupId>io.openshift.booster</groupId>
+  <name>Spring Boot - Istio Distributed Tracing Booster</name>
+  <description>Spring Boot - Istio Distributed Tracing Booster</description>
+
+
+  <properties>
+    <java.version>1.8</java.version>
+
+    <skip.fmp>true</skip.fmp>
```

Comment:
Outside of the scope of this PR as well but we need to get to the bottom of this, once and for all.